tests/handlers/asgi/test_validations.py

```diff
@@ -11,29 +11,31 @@
 
 
 def test_asgi_handler_validation() -> None:
+    app = Litestar()
+
     async def fn_without_scope_arg(receive: "Receive", send: "Send") -> None:
         pass
 
     with pytest.raises(ImproperlyConfiguredException):
-        asgi(path="/")(fn_without_scope_arg).on_registration()
+        asgi(path="/")(fn_without_scope_arg).on_registration(app)
```

Comment:
```suggestion
        asgi(path="/")(fn_without_scope_arg).on_registration(Litestar())
```

For the purpose of isolation. Applies to the other tests as well.